### PR TITLE
Feature/Import Indicators tab from upc branch

### PR DESCRIPTION
**Pull request by nshandra**
### References

- No specific issue for this task, parent issue: [Incorporate information about individual analysis in WebApp HMIS Dictionary](https://app.clickup.com/t/3771yxt)

### Description
- Imports the Indicators tab to the app, the programs tab changes are ignored except the link code.
- The link functionality add a new dependency: [angular-sessionstorage](https://classic.yarnpkg.com/en/package/angular-sessionstorage)

### CODE REVIEW

File: app/datasets/datasets.controllers.js [hunk 1]
```diff
@@ -55,6 +65,33 @@ datasetsModule.controller("datasetsMainController", [
...
+
+        if (sessionStorage.getItem("dataSetName") !== null) {
+            $scope.dataSetName = sessionStorage.getItem("dataSetName");
+            sessionStorage.clear();
+
+            $scope.datasets = datasetsLinkFactory.get({ displayName: $scope.dataSetName }, function () {
+                endLoadingState(true);
+            });
+        }
+        /*else
```
Comment (adrianq) [inline app/datasets/datasets.controllers.js]: Can this comment be deleted?

File: app/dossierIndicators/dossierIndicators.services.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
+//Récupération de tous les indicateurs : phase d'initialisation
```
Comment (adrianq) [inline app/dossierIndicators/dossierIndicators.services.js]: Please translate or remove this comment

File: app/dossierIndicators/dossierIndicators.services.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                    method: "GET",
+                    isArray: false,
+                },
+            }
+        );
+    },
+]);
+
+var qryIndicatorCategoryOptionCombos =
+    dhisUrl + "categoryOptionCombos.json?filter=id\\:eq\\::id&fields=name&paging=false"; //Modification de displayName par Name
```
Comment (adrianq) [inline app/dossierIndicators/dossierIndicators.services.js]: Please remove this comment as well

File: yarn.lock [hunk 4]
```diff
@@ -69,6 +69,11 @@
   version "4.5.12"
   resolved "https://codeload.github.com/tinymce/tinymce-dist/tar.gz/3d007282ded510e32c638b2ed5a9ffe1f219d329"
 
+angular-sessionstorage@^1.1.5:
```
Comment (adrianq) [inline yarn.lock:87]: Do we actually need this new dependency?

File: app/dossierPrograms/dossierPrograms.services.js [hunk 5]
```diff
@@ -473,3 +473,35 @@ searchModule.factory("dossiersProgramVisualizationTableFactory", [
         };
     },
 ]);
+
```
Comment (adrianq) [inline app/dossierPrograms/dossierPrograms.services.js:476]: If this code is not relevant please remove it

**Review by adrianq — CHANGES_REQUESTED** [review]
@nshandra I am aware this code is a merge from a different fork but I don't want to merge it as is. Can you please have a look at the comments below? I am particularly interested in understanding if we can avoid adding a new dependency. I will ask @tokland to review it later on

File: app/datasets/datasets.controllers.js (hunk 1 above)
Comment (nshandra) [reply app/datasets/datasets.controllers.js]: Done.

File: app/dossierIndicators/dossierIndicators.services.js (hunk 2 above)
Comment (nshandra) [reply app/dossierIndicators/dossierIndicators.services.js]: Done. Removed as it wasn't very informative.

File: app/dossierIndicators/dossierIndicators.services.js (hunk 3 above)
Comment (nshandra) [reply app/dossierIndicators/dossierIndicators.services.js]: Removed.

File: app/dossierPrograms/dossierPrograms.services.js (hunk 5 above)
Comment (nshandra) [reply app/dossierPrograms/dossierPrograms.services.js:476]: Done.

File: yarn.lock (hunk 4 above)
Comment (nshandra) [reply yarn.lock:87]: Right now its used for the "link to" functionality in the indicators dossier. I'll look into a possible alternative.

**Review by tokland — APPROVED** [review]

**Pull request merged**